main.go

```diff
@@ -55,6 +55,9 @@ Options:
                          If you want to pass several options to 'useradd', specify
                          one flag '-g' with quoted argument.
                          Like that: '-g "-m -Gwheel"'. [default: -m].
+  -D --delete           Delete user if it does not exists in shadowd.
+  --userdel <args>      Additional parameters for 'userdel' flag when deleting user.
+                         Work like '--useradd' flag for 'useradd'.
```

Comment:
It may be worth to add the flag -r (remove home directory and mail spool) by default?